src/app/(nextjs_migration)/components/client/ExposureCard.tsx

```diff
@@ -0,0 +1,186 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { ReactElement, use, useState } from "react";
+import styles from "./ExposureCard.module.scss";
+import { StatusPill } from "./StatusPill";
+import { StaticImageData } from "next/image";
+import {
+  ChevronDown,
+  EmailIcon,
+  LocationPin,
+  MultipleUsers,
+  OpenInNew,
+  PhoneIcon,
+} from "./Icons";
+import Image from "next/image";
+import { Button } from "./Button";
+import Email from "next-auth/providers/email";
+
+export type Props = {
+  exposureImg: StaticImageData;
+  exposureName: string;
+  exposureType: string;
+  exposureDetailsLink: string;
+  dateFound: string;
+  statusPillType: string;
+  statusPillContent: string;
+};
+
+type DetailsFoundProps = {
+  whichExposed: string; // family | email | phone | address
+  num: number;
+  icon: ReactElement;
+};
+
+export const ExposureCard = (props: Props) => {
+  const {
+    exposureImg,
+    exposureName,
+    exposureType,
+    exposureDetailsLink,
+    dateFound,
+    statusPillContent,
+    statusPillType,
+  } = props;
+
+  const [detailsOpen, setDetailsOpen] = useState(false);
+
+  const DetailsFoundItem = (props: DetailsFoundProps) => {
+    let headline, description;
+    if (props.whichExposed === "family") {
+      headline = "Family members";
+
+      description = `We found ${props.num} family member`;
+      if (props.num > 1) {
+        description = `We found ${props.num} family members`;
+      }
+    }
+
+    if (props.whichExposed === "email") {
+      headline = "Email";
+
+      description = `We found ${props.num} email address`;
+      if (props.num > 1) {
+        description = `We found ${props.num} email addresses`;
+      }
+    }
+
+    if (props.whichExposed === "phone") {
+      headline = "Phone number";
+
+      description = `We found ${props.num} phone number`;
+      if (props.num > 1) {
+        description = `We found ${props.num} phone numbers`;
+      }
+    }
+
+    if (props.whichExposed === "address") {
+      headline = "Address";
+
+      description = `We found ${props.num} address`;
+      if (props.num > 1) {
+        description = `We found ${props.num} addresses`;
+      }
+    }
+
+    return (
+      <>
+        <dt>
+          <span className={styles.exposureTypeIcon}>{props.icon}</span>
+          {headline}
+        </dt>
+        <dl>{description}</dl>
+      </>
+    );
+  };
+  const elementCard = (
+    <div>
```

Comment:
nit: Do we need this wrapper `div`, or can we just use the child `<div className={styles.exposureCard}>` as the top level container?